src/components/Picker.vue

```diff
@@ -262,6 +316,27 @@ export default {
       // Vue 3 does not support $refs as array.
       return component
     },
+    updatePreviewEmoji() {
+      if (this.searchEmojis) {
```

Comment:
❓ Mabye it is possible to just have one `emojis` list and do `this.emojis = this.data.search(...)` when we activate the search?

There is a `if (this.searchEmojis) ...` logic spread over the code, having one `emojis` property would simplify that.
At least, a computed field should be posbbile:

```
   emojis() {
      if (this.searchEmojis) {
          return this.searchEmojis
      } else {
          return this.emojis
      }
```